app/src/main/java/org/mozilla/fenix/tabstray/browser/AbstractBrowserTabViewHolder.kt

```diff
@@ -130,7 +130,8 @@ abstract class AbstractBrowserTabViewHolder(
     private fun updateTitle(tab: TabSessionState) {
         val title = tab.content.title.ifEmpty {
             tab.content.url
-        }
+                .take(MAX_URI_LENGTH)
```

Comment:
Could we use `toShortUrl` here? https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt#66

Reply:
To be more specific: https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/ext/String.kt#28

Reply:
Already answered in the opening comment. Let me know if you want an expanded explanation.